cmd/deploy/deploy.go

```diff
@@ -327,6 +326,9 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		return err
 	}
 
+	// We are doing some operations regarding to manifest load, so if it fails, it should belong to Load Manifest stage
+	oktetoLog.SetStage("")
```

Comment:
This is moved a bit below just because `setDeployOptionsValuesFromManifest` function is doing some validation on the manifest, so if it fails, it should belong to `Load manifest` stage